src/main/java/com/es/phoneshop/dao/AbstractDao.java

```diff
@@ -33,14 +34,14 @@ public void setId(long id) {
         this.id = id;
     }
 
-    public T getItem(Long id){
+    public T getItem(Long id) {
```

Comment:
Don't duplicate implementation of getItem in ProductDao.
getItem method should be common for ProductDao and OrderDao